app/shared/productTypes.tsx

```diff
@@ -485,6 +503,7 @@ export const ProductTypes: { [productKey in ProductTypeKeys]: ProductType } = {
     productPage: "subscriptions"
   },
   guardianweekly: {
+    productTitle: calculateProductTitle("Guardian Weekly"),
```

Comment:
this should be...
```suggestion
    () => "Guardian Weekly",
```
... as we never want to display the plan name as part of the product title - I felt sure we did this though when we removed the `isSixForSix` from this call 🤔